wallet/src/types.rs

```diff
@@ -238,6 +238,8 @@ pub enum Event {
     /// A list of hashes of blocks that are now considered orphaned.
     BlocksOrphan(Vec<String>),
     /// A new movement (transaction) affecting balance.
+    LocalTransactionSent(model::BalanceMovement),
```

Comment:
Should we create a new event, or should we reuse the existing `Movement`?

Reply:
Depends on whether it makes a difference for sheikah